.azuredevops/platformPipelines/platform.deployment.removal.yml

```diff
@@ -0,0 +1,125 @@
+name: '.Platform - Remove Deployments'
+
+parameters:
+  - name: handleSubscriptionScope
+    displayName: Include Subscription deployments
+    type: boolean
+    default: true
+  - name: handleManagementGroupScope
+    displayName: Include Management Group deployments
+    type: boolean
+    default: true
+  - name: maxDeploymentRetentionInDays
```

Comment:
So we're not using settings.yml for that, but leveraging the pipeline input instead?

Reply:
Yes. It was quite annoying without. Now it would be used similar to the removal flag for the module pipelines